website/content/en/docs/releases/deprecated.md

```diff
@@ -5,10 +5,9 @@ weight: 10
 
 The following features are deprecated:
 
-- CentOS 7 support
-- Loading non-strict YAMLs (i.e., YAMLs with unknown properties)
-- `limactl show-ssh` command (Use `ssh -F ~/.lima/default/ssh.config lima-default` instead)
-- Ansible provisioning mode (Use `ansible-playbook playbook.yaml` after the start instead)
+- `limactl show-ssh` command: deprecated in v0.18.0 (Use `ssh -F ~/.lima/default/ssh.config lima-default` instead)
+- Ansible provisioning mode: deprecated in Lima v1.1.0 (Use `ansible-playbook playbook.yaml` after the start instead)
+- `limactl --yes` flag: deprecated in Lima v2.0.0 (Use `limactl (clone|rename|edit|shell) --start` instead)
```

Comment:
Should mention `template://` URLs because they generate a warning now.

Reply:
One reason to document it as deprecated is to justify the noise you get when you use the old form: "If it is not deprecated, why does it generate a warning?"

And I think we should be able to remove it in Lima 3.0. Not sure if we will, because it is not blocking anything, but we could.